protocol-definitions/Jet.yaml

```
@@ -301,3 +301,23 @@ methods:
           nullable: false
           since: 2.5
           doc: ''
+  - id: 16
+    name: isJobUserCancelled
+    since: 2.6
+    doc: ''
+    request:
+      retryable: true
+      partitionIdentifier: -1
+      params:
+        - name: jobId
+          type: long
+          nullable: false
+          since: 2.6
+          doc: ''
+    response:
+      params:
+        - name: response
+          type: boolean
+          nullable: false
+          since: 2.6
+          doc: ''
```

nit: no new line at end of file
```suggestion
          doc: ''

```